### ISSUE TITLE
Header displayed twice in Bootstrap and HTML/CSS Waypoints

### ISSUE THREAD

**Issue by sanspace** [FreeCodeCamp/FreeCodeCamp#3769]
![header-twice](https://cloud.githubusercontent.com/assets/347714/10540330/a0e7f868-7424-11e5-824a-9321c1ec90f3.png)

I often see the header displayed twice in the output area. I noticed this in HTML/CSS and Bootstrap waypoints so far.

cc: @QuincyLarson

**Comment by sanspace** [comment]
In the Results iframe, the contents are there at the start of the <body> and also before closing of </body>. I'm not sure how it got there twice. Ideally this should be only once.

**Comment by BerkeleyTrue** [comment]
Closing as duplicate of #2706

Please see https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/Searching-for-Your-Issue-on-Github

Happy Coding!

**Comment by sanspace** [comment]
I did try searching for it. I couldn't find the original issue. May be I should improvise my search skills! :(
